godtools/Managers/LanguagesManager.swift

```diff
@@ -19,6 +19,21 @@ class LanguagesManager: GTDataManager {
     
     var selectingPrimaryLanguage = true
     
+    static var _defaultLanguage: Language?
+    static var defaultLanguage: Language? {
```

Comment:
I see that this gets set when the getter is called for the first time. How does it ever get changed though? I never see the setter called.